src/main/java/nl/novi/TechItEasy/controllers/OutOfBoundExceptionHandler.java

```diff
@@ -0,0 +1,15 @@
+package nl.novi.TechItEasy.controllers;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
```

Comment:
Het is niet nodig (of handig) om 2 `@`ControllerAdvice klassen te hebben. Je kunt in zo'n klasse al je `@`ExceptionHandler methodes zetten.